arm-trafficmanager/2017-05-01/swagger/trafficmanager.json

```diff
@@ -0,0 +1,972 @@
+          "Endpoints"
+        ],
+        "operationId": "Endpoints_Delete",
+        "description": "Deletes a Traffic Manager endpoint.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "endpointType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The type of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "name": "endpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "The Traffic Manager Endpoint was deleted successfully.",
+             "schema": { 
+               "$ref": "#/definitions/DeleteOperationResult" 
+             } 
+          },
+          "204": { 
+             "description": "The Traffic Manager Endpoint does not exist. It could have been deleted on a previous request.",
+             "schema": { 
+               "$ref": "#/definitions/DeleteOperationResult" 
+             } 
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Network/checkTrafficManagerNameAvailability": {
+      "post": {
+        "tags": [
+          "Profiles"
+        ],
+        "operationId": "Profiles_CheckTrafficManagerRelativeDnsNameAvailability",
+        "description": "Checks the availability of a Traffic Manager Relative DNS name.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CheckTrafficManagerRelativeDnsNameAvailabilityParameters"
+            },
+            "description": "The Traffic Manager name parameters supplied to the CheckTrafficManagerNameAvailability operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
```

Comment:
I am bit confused here. If you are defining a parameter referenced as `SubscriptionIdParameter ` in `parameters` section which is by [definition](https://github.com/Azure/azure-rest-api-specs/pull/1180/files/b1f877954e737e88c943c184dd41a394b007ab29#diff-09eb574a88e842533ca4d6116eb9516aR957) is `"in": "path"` & `"required": true` and REST path that `/providers/Microsoft.Network/checkTrafficManagerNameAvailability` does not have that parameters at all. How are we going to take that as input? 

Could you please help me by providing example curl / fiddler example targeting this path and also taking `SubscriptionIdParameter ` as part of it, to better understand it?